proto/minder/v1/minder.proto

```diff
@@ -100,12 +100,14 @@ message Cursor {
         (buf.validate.field).string = {
             pattern: "^[[:word:]=]*$",
             max_len: 200,
-        }
+        },
+        (google.api.field_behavior) = REQUIRED
```

Comment:
is the cursor actually required? Thought an empty cursor was valid and it would be the start.

Reply:
You're right, thank you